samples/apps/copilot-chat-app/webapi/Skills/SemanticChatMemoryItem.cs

```diff
@@ -1,4 +1,7 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+﻿// TODO move memory items to different directory
```

Comment:
What does this TODO mean? This is also in SemanticChatMemory.cs.

Reply:
removed